client/src/common/components/CardItem.jsx

```diff
@@ -57,7 +58,7 @@ class CardItem extends PureComponent {
         role="figure"
       >
         <header className="card-item__header">
-          {isPrivate && <LockIcon />}
+          {type === ListType.LIMITED && <LockIcon />}
```

Comment:
I think it would be better to store this in variable above return
```
const isLimitedList = type === ListType.LIMITED;

return (........
{isListLimited && <LockIcon />}
```